.github/workflows/release.yml

```diff
@@ -63,4 +63,4 @@ jobs:
             
             Created by https://github.com/mislav/bump-homebrew-formula-action
         env:
-          COMMITTER_TOKEN: ${{ secrets.PUBLIC_REPO_ACCESS_TOKEN }}
+          COMMITTER_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
per discussions, i dont think this would work either as it needs a token to a homebrew-core fork.

Reply:
I think it gets forked automatically. See the atmos prs that fork using the cloudpossebot

Example https://github.com/Homebrew/homebrew-core/pull/119212

Reply:
@chenrui333 are you saying that we need to use a different token with more rights? Or that runatlantis org needs to create a homebrew-core fork?

What do you think the next steps are to ensure this works as expected?

Reply:
runatlantis need to have a homebrew-core fork. And the token need to have write permission to do it.

We can discuss about it in the monthly meeting.

Reply:
Ok. I have disabled this for now to allow green checks upon release.